runtime/service_integ_test.go

```diff
@@ -2032,6 +2032,93 @@ func TestCreateVM_Isolated(t *testing.T) {
 	}
 }
 
+func TestPauseResumeVM_Isolated(t *testing.T) {
```

Comment:
I'd like to have some other cases here.

- If we pause a VM multiple times, does it work? Does Firecracker itself returns an error that callers can understand? Should firecracker-containerd handle the case specifically by having some flags?
- If we resume the VM without pausing, does it work? The same questions above would be applied here as well.

Reply:
sorry for the delay, we will try to add the tests shortly